app/controllers/help_controller.rb

```diff
@@ -1,4 +1,8 @@
 class HelpController < ApplicationController
-	before_filter -> { @css_framework = :bootstrap }
-	layout ->(c) { request.format == :mobile ? "application" : "with_header_with_footer" }
-end
+  before_filter -> { @css_framework = :bootstrap }
+  layout -> (c) { request.format == :mobile ? "application" : "with_header_with_footer" }
+
+  def faq
+    gon.preloads[:chat_enabled] = AppConfig.chat.enabled ? 'enabled' : 'disabled'
```

Comment:
Here you use the key `gon.preloads[:chat_enabled]` but in the JS you use the key `gon.chat_enabled`. I find the key `gon.chat_enabled` better, so chance this instance.